README.md

```diff
@@ -77,5 +77,5 @@ find build -iname "*.mp4"
 
 ## Authors
 
-- [Leo Stein](https://duetosymmetry.com/) ([@duetosymmetry](https://github.com/duetosymmetry))
 - [Matthew Feickert](http://www.matthewfeickert.com/) ([@matthewfeickert](https://github.com/matthewfeickert))
+- [Leo C. Stein](https://duetosymmetry.com/) ([@duetosymmetry](https://github.com/duetosymmetry))
```

Comment:
> Authors ordered by amount of work done here

I'd say that the creation and thought originally put into this gets top billing, but I'll defer to your choices here. :slightly_smiling_face: